### PR TITLE
Chore: created find helpers

### PR DESCRIPTION
**Pull request by galmail**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
   "scripts": {
     "clean": "rimraf build",
     "start": "yarn test",
-    "start:watch": "yarn test --watch",
+    "start:watch": "yarn test --watch --reporters=\"default\"",
```
Comment (jensdev) [inline package.json:16]: Why is this?

File: package.json [hunk 2]
```diff
@@ -69,6 +69,16 @@
     "outputDirectory": "test-results",
     "outputName": "jest-junit.xml"
   },
+  "jest-stare": {
```
Comment (jensdev) [inline package.json:72]: Why is this?

File: src/services/data/comments.ts [hunk 3]
```diff
@@ -62,6 +62,51 @@ export default (client, httpAuth: HttpInstance) => ({
...
+   * @param schemaId the schema Id
+   * @param documentId the document Id
+   * @param rql an optional rql string
+   * @returns the first element found
+   */
+  async findById(
+    id: ObjectId,
+    schemaId: ObjectId,
+    documentId: ObjectId,
+    rql?: RQLString
```
Comment (jensdev) [inline src/services/data/comments.ts:77]: Optional parameters should go in seperate object.

File: src/services/data/comments.ts [hunk 4]
```diff
@@ -62,6 +62,51 @@ export default (client, httpAuth: HttpInstance) => ({
...
+  ): Promise<Comment> {
+    const rqlWithId = rqlBuilder(rql).eq('id', id).build();
+    const res = (
+      await client.get(
```
Comment (jensdev) [inline src/services/data/comments.ts:81]: Any preference to do it like vs for example something like:
```ts
  async findById(id: ObjectId, rql?: RQLString): Promise<Schema> {
    const rqlWithId = rqlBuilder(rql).eq('id', id).build();
    const res = await this.find({ rql: rqlWithId});
    return res.data[0];
  },
```

File: src/services/dispatchers/index.ts [hunk 5]
```diff
@@ -15,8 +15,10 @@ export const dispatchersService = (
     basePath: DISPATCHERS_BASE,
   });
 
+  const dispatchersMethods = dispatchers(client, httpWithAuth);
+
   return {
-    ...dispatchers(client, httpWithAuth),
+    ...dispatchersMethods,
```
Comment (jensdev) [inline src/services/dispatchers/index.ts:21]: Any reason why this is changed?

File: package.json (hunk 1 above)
Comment (galmail) [reply package.json:16]: I had to change it, cause the tests were going into infinite loop!!

File: src/services/data/comments.ts (hunk 3 above)
Comment (galmail) [reply src/services/data/comments.ts:77]: I wasn't sure if we should... as the find helpers should be kept simple.

File: src/services/data/comments.ts (hunk 4 above)
Comment (galmail) [reply src/services/data/comments.ts:81]: that's what I wanted to do, but unfortunately `this` keyword refers to the calle.

File: CHANGELOG.md [hunk 6]
```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.1.0]
+
+- Added `findById`, `findByName` and `findFirst` methods to services
+
+## [3.0.1]
```
Comment (jensdev) [inline CHANGELOG.md:14]: Remove this one

File: src/services/data/comments.ts (hunk 4 above)
Comment (galmail) [reply src/services/data/comments.ts]: all done! thanks!

File: src/services/data/comments.ts (hunk 3 above)
Comment (galmail) [reply src/services/data/comments.ts]: done!

File: CHANGELOG.md (hunk 6 above)
Comment (galmail) [reply CHANGELOG.md:10]: ok done

**Review by jensdev — APPROVED** [review]

**Pull request merged**